src/searcher_frame_document.rs

```diff
@@ -44,6 +44,13 @@ impl SearchResult {
         Ok(s)
     }
 
+    #[getter]
+    fn unique_sentences(&self, py: Python) -> PyResult<BTreeSet<u64>> {
```

Comment:
can we prepend our custom stuff with kapiche, so its easy to see our customisations

Reply:
Yeah, I need to refactor all of this eventually at some point.  The main reason why this won't work upstream as-is is because it is currently hard-coded to recognize specific fast fields for document, frame and sentence.  This makes it very Kapiche-specific. In the future I would like to refactor this code so that the specific names of fast fields being included can be project-dependent (remove the hardcoding). Then it could be used upstream.